src/config/session-management.md

```diff
@@ -40,3 +40,14 @@ service, as waiting for a D-Bus activation can lead to issues.
 There is an alternative D-Bus configuration which takes advantage of elogind for
 features such as seat detection. It requires installing the `dbus-elogind`,
 `dbus-elogind-libs` and `dbus-elogind-x11` packages.
+
+## seatd
+
+[seatd(1)](https://man.voidlinux.org/seatd.1) is a minimal seat management
+daemon and an alternative to elogind.
```

Comment:
technically, it works on top of elogind, too

Reply:
@travankor Care to elaborate on that? I am pretty sure seatd doesn't depend on elogind.

Reply:
https://git.sr.ht/~kennylevinsen/seatd#why-does-libseat-support-elogind

Reply:
seatd != libseat (duh). Furthermore, it seems you absolute don't understand what any of those two do.

Reply:
@ofiala-a51 please be politer when making comments. Misunderstandings can happen.

travankor probably thought it would be worth mentioning libseat, and, even if they were confused, there's no reason to be rude about it.

Reply:
Well, it's not a good idea to go "reviewing" a PR if you don't understand the topic. I agree I should have reacted better though. My apologies.